src/Microsoft.OData.Core/UriParser/SemanticAst/SelectExpandClauseExtensions.cs

```diff
@@ -86,7 +86,16 @@ internal static void GetSelectExpandPaths(this SelectExpandClause selectExpandCl
         /// <returns>String list generated from selected items</returns>
         internal static List<string> GetCurrentLevelSelectList(this SelectExpandClause selectExpandClause)
         {
-            return selectExpandClause.SelectedItems.Select(GetSelectString).Where(i => i != null).ToList();
+            List<string> levelSelectList = new List<string>(); 
+            foreach (var selectItem in selectExpandClause.SelectedItems)
+            {
+                IList<string> selectList = GetSelectString(selectItem);
+                if (selectList != null)
+                {
+                    levelSelectList.AddRange(selectList);
+                }
+            }   

```

Comment:
>    [](start = 13, length = 3)

remove the whitespace? and add online after }

Reply:
Resolved while refactoring